core/src/main/scala/org/apache/spark/deploy/JsonProtocol.scala

```diff
@@ -312,9 +312,9 @@ private[deploy] object JsonProtocol {
     ("waitingDrivers" -> obj.activeDrivers.count(_.state == DriverState.SUBMITTED)) ~
     ("cores" -> cores) ~
     ("coresused" -> coresUsed) ~
-    ("coresutilization" -> 100 * coresUsed / cores) ~
+    ("coresutilization" -> (if (cores == 0) 100 else 100 * coresUsed / cores)) ~
```

Comment:
If `cores` is 0, why `utilization` isn't 0?

Reply:
This utilization aims to inform cluster scale up. So, `0` should be `100%` used.